docs/about-filecoin/primer.md

```diff
@@ -0,0 +1,147 @@
+
+Length: 1 hour
+
+Supplemental Materials:
+
+- [Announcing lotus, our first alternate Filecoin implementation](https://filecoin.io/blog/announcing-lotus/)
+- [Filecoin Docs: lotus and go-ipfs interactions](https://docs.filecoin.io/build/examples/network-inspector/lotus-and-go-ipfs-interactions/)
+- [How to set up the lotus and go-ipfs API clients](https://docs.filecoin.io/build/examples/network-inspector/step-3-set-up-the-lotus-and-go-ipfs-api-clients/)
+
+### [Textile’s tools for the Filecoin network](https://www.youtube.com/watch?v=IZ8M9m9_uJY)
+
+Textile is a set of open-source tools that provide a decentralized database, IPFS-based storage, content hosting, and more, over libp2p, IPFS, and Filecoin. This workshop covers how to use Textiles Threads, Buckets, and Hub tools to build apps for the Filecoin network (focus on JavaScript). The session includes an introduction to Textile’s suites of tools, basic command-line (CLI) actions, and Textile’s JavaScript Client(s). To prepare, we encourage you to install a few things before we get started:
+
+- [Install](https://docs.textile.io/hub/accounts/) the Textile Hub command line interface (CLI) on your computer (so that you can connect to Textile’s Hub via the Hub CLI tools).
+- [Set up Node.js](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs) on your computer.
+
+Length: 1 hour 20 minutes
+
+Supplemental Materials:
+
+- [A protocol & event-sourced database for decentralized user-siloed data](https://blog.textile.io/introducing-textiles-threads-protocol/)
+- [Want to try Filecoin? Textile is building tools to make it as easy as 123](https://blog.textile.io/developer-tools-for-filecoin-ipfs-web/)
+- [Developer Tutorial: Simple Pinning Service Demo](https://www.youtube.com/watch?v=mQF0o2IFhVI)
+- [How to build a Meme Marketplace using React, Textile Hub, and MetaMask](https://www.youtube.com/watch?v=UaTr0JSg4ZQ)
+
+### [Intro to Textile’s Powergate](https://www.youtube.com/watch?v=aiOTSkz_6aY)
+
+Textile’s Powergate is an API-driven solution for deploying multitiered storage across Filecoin and IPFS. Persistent storage on Filecoin allows rich storage configuration for data, such as replication factor, miner selection, deal renewal, and repair. Network available storage is configurable and provided through a connected IPFS peer or pinning network.
+
+Length: 35 minutes
+
+Supplemental Materials:
+
+- [Filecoin Docs: Powergate](https://docs.filecoin.io/build/tools/powergate/)
+- [Filecoin Docs: How to build a simple pinning service using Powergate](https://docs.filecoin.io/build/examples/simple-pinning-service/overview/)
+
+### [Intro](https://www.youtube.com/watch?v=FJjPMKRy8xQ) to [Slate](https://www.youtube.com/watch?v=Rknj2GqvJtg)
+
+[Building an App with Filecoin from scratch - using Slate Components & Powergate](https://www.youtube.com/watch?v=FJjPMKRy8xQ)
```

Comment:
This corresponds to https://docs.filecoin.io/build/examples/simple-pinning-service/overview/ ? Should be linked then.